jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebAppConfiguration.java

```diff
@@ -32,14 +32,10 @@ public WebAppConfiguration()
         addDependencies(WebXmlConfiguration.class, MetaInfConfiguration.class, WebInfConfiguration.class);
         addDependents(JettyWebXmlConfiguration.class);
         protectAndExpose(
-            "org.eclipse.jetty.util.log.",
-            "org.eclipse.jetty.server.session.SessionData",
-            "org.eclipse.jetty.servlet.StatisticsServlet", 
+            "org.eclipse.jetty.servlet.StatisticsServlet",
             "org.eclipse.jetty.servlet.DefaultServlet", 
-            "org.eclipse.jetty.servlet.NoJspServlet",
-            "org.eclipse.jetty.continuation.");
-        expose( // TODO Evaluate why these are not protectAndExpose?
-            "org.eclipse.jetty.servlet.listener.",
-            "org.eclipse.jetty.alpn.");
+            "org.eclipse.jetty.servlet.NoJspServlet"
+        );
+        expose("org.eclipse.jetty.servlet.listener.");
```

Comment:
All these are classes/packages are in `org.eclipse.jetty.servlet`.
Would it not make more sense to have a `Configuration` class in the `jetty-servlet` module rather than in `jetty-webapp`?

Reply:
Not really.  servlet is usable without webapps.    But configuration really about configuring webapps - specifically by a combination of  convention and the configurations we push into it from our modules.   If we wanted to move Configuration down to servlet, then we'd have to move the WebAppClassLoader (or at least an abstraction of it) down as well... and then we start having no difference between servlet and webapp.

tl;dr; any direct users of jetty-servlet configure by using API directly and probably don't have a different classloader (maybe not even a context).